projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order/checkout/delivery-address.ts

```diff
@@ -6,17 +6,14 @@ import { TabElement } from '../../tabbing-order.model';
 
```

Comment:
Random:
be sure to remove this task from todo _after_ we merge the schematics PR to epic (to avoid conflicts)